autogpt/commands/execute_code.py

```diff
@@ -21,17 +22,25 @@ def execute_python_file(filename: str, config: Config) -> str:
     Returns:
         str: The output of the file
     """
-    logger.info(f"Executing file '{filename}'")
+    logger.info(
+        f"Executing python file '{filename}' in working directory '{CFG.workspace_path}'"
+    )
 
     if not filename.endswith(".py"):
         return "Error: Invalid file type. Only .py files are allowed."
 
     if not os.path.isfile(filename):
```

Comment:
Nit: this does not enforce `restrict_to_workspace`. Would you mind amending it to use `Workspace.get_path`?

Reply:
Oh, this is code that already existed. I didn't change it on purpose to limit regressions.

Reply:
I know it already existed. It should be fixed regardless, and I think it fits within the scope given by the title of this PR. :)

Reply:
OK I added it. Not sure if the way I instantiated a `Workspace` is correct. Would be nice to be able to grab the agent's workspace.